base/distro.bzl

```
@@ -1,5 +1,7 @@
 load("@package_bundle_amd64_debian9//file:packages.bzl", packages_amd64_debian9 = "packages")
 load("@package_bundle_amd64_debian10//file:packages.bzl", packages_amd64_debian10 = "packages")
+load("@package_bundle_arm64_debian9//file:packages.bzl", packages_arm_debian9 = "packages")
+load("@package_bundle_arm64_debian10//file:packages.bzl", packages_arm_debian10 = "packages")
```

Should it be like this?
```suggestion
load("@package_bundle_arm_debian9//file:packages.bzl", packages_arm_debian9 = "packages")
load("@package_bundle_arm_debian10//file:packages.bzl", packages_arm_debian10 = "packages")
```

---

good catch 😅